src/js/components/AppHealthBarWithTooltipComponent.jsx

```diff
@@ -0,0 +1,79 @@
+var React = require("react/addons");
+
+var TooltipMixin = require("../mixins/TooltipMixin");
+var AppHealthBarComponent = require("./AppHealthBarComponent");
+var AppHealthBreakdownComponent = require("./AppHealthBreakdownComponent");
+var Util = require("../helpers/Util");
+var HealthStatus = require("../constants/HealthStatus");
+
+var appHealthBreakdownFields = [
+  {label: "Healthy", key: "healthy", state: HealthStatus.HEALTHY},
+  {label: "Unhealthy", key: "unhealthy", state: HealthStatus.UNHEALTHY},
+  {label: "Unknown", key: "unknown", state: HealthStatus.UNKNOWN}
+];
```

Comment:
Oh yeah, if you're referring to https://github.com/mesosphere/marathon-ui/blob/feature/2657-breakdown-tooltips/src/js/components/AppHealthBarComponent.jsx#L10-L17 – makes perfect sense. Also in light of @aldipower 's comment https://github.com/mesosphere/marathon/issues/2657#issuecomment-160595504